src/xbps/repositories/index.md

```diff
@@ -8,7 +8,20 @@ which may also be signed.
 Note that, while local repositories do not require signatures, remote
 repositories *must* be signed.
 
-## Official Repositories
+## The main repository
+
+The locations of the main repository in relation to a base [mirror
+URL](./mirrors/index.md) are:
+
+| System type | Location           |
+|-------------|--------------------|
+| glibc       | `/current`         |
+| musl        | `/current/musl`    |
+| aarch64     | `/current/aarch64` |
+
+Note: musl binaries are not available for i686.
```

Comment:
Do we need to have this info?

Reply:
This came from the existing "Installation via chroot" page; if you think it can be removed, i'm happy to do so. :-)

Reply:
I'm not entirely sure. It is documented in the musl page already.

Reply:
Okay, removing.